.travis.yml

```diff
@@ -7,13 +7,25 @@ env:
   matrix:
   - CI_TEST: ci-fast
     CI_SCALA_VERSION: 2.11.11
+    CI_SBT_VERSION: 0.13.16
```

Comment:
Do we need this line if we don't use it in ci-fast?

Reply:
see https://github.com/scalacenter/scalafix/pull/449/files#diff-683680af9315ad6f6396cda09246a992R12